suite-common/wallet-core/src/send/sendFormStellarThunks.ts

```diff
@@ -0,0 +1,264 @@
+import {
+    TransactionBuilder,
+    Account,
+    Networks,
+    Memo,
+    Operation,
+    Asset,
+} from '@stellar/stellar-sdk';
+
+import { BigNumber } from '@trezor/utils/src/bigNumber';
+import TrezorConnect, { FeeLevel } from '@trezor/connect';
+import { transformTransaction } from '@trezor/connect-plugin-stellar';
+import {
+    calculateTotal,
+    calculateMax,
+    getExternalComposeOutput,
+    formatNetworkAmount,
+} from '@suite-common/wallet-utils';
+import {
+    PrecomposedLevels,
+    PrecomposedTransaction,
+    ExternalOutput,
+    PrecomposedTransactionStellar,
+} from '@suite-common/wallet-types';
+import { createThunk } from '@suite-common/redux-utils';
+
+import {
+    ComposeTransactionThunkArguments,
+    ComposeFeeLevelsError,
+    SignTransactionThunkArguments,
+    SignTransactionError,
+} from './sendFormTypes';
+import { SEND_MODULE_PREFIX } from './sendFormConstants';
+
+// Copied from ./sendFormRippleThunks.ts
```

Comment:
would be great to make duplicates generic

Reply:
We reused most of the code in `composeRippleStellarTransactionFeeLevelsThunk`.

https://github.com/trezor/trezor-suite/pull/16483/files#diff-d247a3ab9d02c4080fe88820c1fc6199254031ad9ab163aebace44ddc8f71367R102